Fix:

error: call to undeclared library function 'finite' with type 'int (double)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]

https://github.com/astromatic/sextractor/issues/24
https://github.com/astromatic/sextractor/issues/27
https://github.com/astromatic/sextractor/commit/ced65570cb5b7073361dbf2c3c60631c3f54d0f9
--- a/src/levmar/compiler.h
+++ b/src/levmar/compiler.h
@@ -60,7 +60,12 @@
 #ifdef _MSC_VER
 #define LM_FINITE _finite // MSVC
 #elif defined(__ICC) || defined(__INTEL_COMPILER) || defined(__GNUC__)
-#define LM_FINITE finite // ICC, GCC
+#  ifdef __APPLE__
+#  include <math.h>
+#  define LM_FINITE isfinite // Apple
+#  else
+#  define LM_FINITE finite // ICC, GCC
+#  endif
 #else
 #define LM_FINITE finite // other than MSVC, ICC, GCC, let's hope this will work
 #endif
